app/src/behaviors/behavior_sensor_rotate_key_press.c

```diff
@@ -36,26 +73,22 @@ static int on_sensor_binding_triggered(struct zmk_behavior_binding *binding,
 
     switch (value.val1) {
     case 1:
-        keycode = binding->param1;
+        keycode_position = 1;
         break;
     case -1:
-        keycode = binding->param2;
+        keycode_position = 2;
         break;
     default:
         return -ENOTSUP;
     }
 
-    LOG_DBG("SEND %d", keycode);
-
-    ZMK_EVENT_RAISE(zmk_keycode_state_changed_from_encoded(keycode, true, timestamp));
-
-    // TODO: Better way to do this?
-    k_msleep(5);
-
-    return ZMK_EVENT_RAISE(zmk_keycode_state_changed_from_encoded(keycode, false, timestamp));
+    zmk_behavior_queue_add(keycode_position, *binding, true, 20);
```

Comment:
This will delay both the press and release by 20 ms instead of immediately pressing and then releasing after 20 ms. Was that intended?